content/messaging/customer-chat-api/changelog/index.mdx

```diff
@@ -40,6 +40,11 @@ This document is the record of all the changes in the **Customer Chat API**, Web
   - the `group_id` parameter was renamed to `id`.
 - In the [**List License Properties**](/messaging/customer-chat-api/v3.3/#list-license-properties) method, the `license_id` parameter was removed.
 
+### Greetings
```

Comment:
Could you place this section below Events so that it's ordered alphabetically? Also, I know it's not in the scope, but  I've noticed that ## Config and ##Localization are h2 instead of h3, could you fix that? 🙏🏻